cmd/bosun/sched/check.go

```diff
@@ -40,7 +40,14 @@ func NewStatus(ak expr.AlertKey) *State {
 	}
 }
 
-func (s *Schedule) Status(ak expr.AlertKey) *State {
+func (s *Schedule) GetStatus(ak expr.AlertKey) *State {
```

Comment:
Should this just return the status to remove the possibility of thinking you can do stuff with it?